store/src/sql/access/stream.rs

```diff
@@ -0,0 +1,147 @@
+use anyhow::anyhow;
+use cid::Cid;
+
+use crate::{
+    sql::entities::{
+        EventHeader, EventHeaderRow, EventType, IncompleteStream, StreamCid, StreamCommitRow,
+        StreamRow,
+    },
+    Error, Result, SqlitePool, SqliteTransaction,
+};
+
+/// Access to the stream and related tables. Generally querying events as a stream.
+pub struct CeramicOneStream {}
```

Comment:
this name makes me think the type represents a single Stream, rather than representing the subsystem for accessing and working with Streams.

Can we call it `CeramicStreamStore` or something like that?

Reply:
Yeah, that makes sense. These access structs were intended to be a sort of grouping of higher level functionality than just single table/entity writes but the names are pretty much copied straight from the tables which is a misnomer for sure.